vagrant/catalog/teacherActions.py

```diff
@@ -0,0 +1,409 @@
+import flask
+from flask import Flask, render_template, request, redirect, url_for, jsonify
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from database_setup import Base, Teacher, Student, Goal, StudentGoalLink, Group, StudentGroupLink
+from sqlalchemy import DateTime
+app = Flask(__name__)
+
+engine = create_engine('sqlite:///teacheractions.db')
+
+Base.metadata.bind = engine
+
+DBSession = sessionmaker(bind=engine)
+session = DBSession()
+@app.route('/loggedin/<int:group_id>/showallstudentsingroup', methods=['GET'])
```

Comment:
instead of showallstudentsingroup the route should just end in /students 

```
@app.route('/loggedin/<int:group_id>/students', methods=['GET'])
```

Remember these are the urls so they should be simple